lisa/sut_orchestrator/ready.py

```diff
@@ -35,6 +45,13 @@ def supported_features(cls) -> List[Type[Feature]]:
             features.SerialConsole,
         ]
 
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        runbook: ReadyPlatformSchema = self.runbook.get_extended_runbook(
```

Comment:
How about just reset the dirty flag in the ready platform? It looks easier to understand.